File: SPOJ/CANDY 3.c
#include<stdio.h>
int main()
{
 unsigned long long int T,n,m,i,s;
 char *p;
 scanf("%llu",&T);
 while(T)
 {
     scanf("%llu",&n);
     s=0;
     i=0;
     while(i<n)
     {
       scanf("%llu",&m);
       s+=m;
       i++;
     }
     if(s%n==0)
        printf("YES\n");
     else
        printf("NO\n");
     T--;
 }
return 0;
}

File: SPOJ/CANDY 3.py
T=input()
while(T):
    m=raw_input()
    n=input()
    s=0
    p=n
    while n:
        m=input()
        s+=m
        n=n-1
    if s%p==0:
        print "YES"
    else:
        print "NO"
    T=T-1
